st2common/st2common/rbac/types.py

```diff
@@ -91,6 +94,7 @@ class PermissionType(Enum):
     EXECUTION_RE_RUN = 'execution_rerun'
     EXECUTION_STOP = 'execution_stop'
     EXECUTION_ALL = 'execution_all'
+    EXECUTION_VIEWS_FILTERS_LIST = 'execution_view_filters_list'
```

Comment:
I'm saying that `VIEW` part is not needed, not that we should remove `INDEX` part. The whole idea of views was added out of sheer necessity due to limitations of pecan we no longer have. That's minor enough, yet kinda annoying.